boards/shields/lmp90100_evb/Kconfig.defconfig

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019 Vestas Wind Systems A/S
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_LMP90100_EVB
+
+if ADC
+
+config GPIO
```

Comment:
GPIO should be enabled by default as part of board configuration (if not the case, board configuration should be fixed), so this should not be needed.

Reply:
@erwango Fixed, thank you.